### DISCUSSION TITLE
OpenDTU Fusion 2.1/2.2 CAN/ISO shield pylontech + huawei

### DISCUSSION THREAD

**Discussion by rudolfp1979** [hoylabs/OpenDTU-OnBattery#1179] [General]
Hat hier jemand schon so ein Setup am laufen mit einem Huawei Ladegerät sowie einem Pylontech und mindestens einem Victron an einer opendtu fusion Platine.  BZW. ist dies überhaupt möglich und falls ja an welche pins würde dann der can des Huawei angeschlossen werden oder besser gesagt falls noch eine solche Platine SBC-CAN01 angeschlossen werden müsste geht das überhaupt zusätzlich zum can/shield. Aktuell verwende ich einen Victron sowie einen Pylontech und lasse über eine Schaltsteckdose ein normales Netzteil bei zuviel Stromverzeugung den Pylontech zusätzlich laden. ich möchte das aber feingranularer regeln können und da kommt so ein huawei dann ins Spiel.

**Comment by Solarteur** [comment]
Den Beitrag hier
https://github.com/helgeerbe/OpenDTU-OnBattery/discussions/1134
gesehen?
